config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -207,8 +218,12 @@ presubmits:
         - --use-logexporter
         image: gcr.io/k8s-testimages/kubekins-e2e:v20200807-c10eea0-master
         resources:
+          limits:
+            cpu: 6
+            memory: "8Gi"
```

Comment:
Is 8GB enough for 2000-node presubmit?

Reply:
IIRC When I was tracking Kubemark manually I've seen an usage of a little above 1 GB. Assuming that it will grow linearly, we can go for with 8GB. Also, for Kubemark 5000 nodes test we are using 16 GB, so again it suggest that we are good with half of that for 2000 nodes.

However, we can go for 'better safe than sorry' way and increase this request, especially that it's an optional presubmit that is rarely triggered. I will change to 16GB.